lighthouse-core/lib/asset-saver.js

```diff
@@ -143,11 +145,14 @@ function prepareAssets(artifacts, audits) {
 function saveAssets(artifacts, audits, pathWithBasename) {
   return prepareAssets(artifacts, audits).then(assets => {
     assets.forEach((data, index) => {
-      const traceData = data.traceData;
       const traceFilename = `${pathWithBasename}-${index}.trace.json`;
-      fs.writeFileSync(traceFilename, JSON.stringify(traceData, null, 2));
```

Comment:
you mentioned the WPT trace parser doesn't do pretty JSON.
Should we start saving it in the right format? (or another PR i suppose)